src/protocols/ssh/input.c

```diff
@@ -88,8 +204,7 @@ int guac_ssh_user_size_handler(guac_user* user, int width, int height) {
     if (ssh_client->term_channel != NULL) {
         pthread_mutex_lock(&(ssh_client->term_channel_lock));
         libssh2_channel_request_pty_size(ssh_client->term_channel,
-                guac_terminal_get_columns(terminal),
-                guac_terminal_get_rows(terminal));
+                terminal->term_width, terminal->term_height);
```

Comment:
This shouldn't change to `terminal->term_width`, etc. which are now internal members of the terminal struct. The getter functions are the public interface.